test/smoke/actionvalidate_test.go

```diff
@@ -26,10 +25,17 @@ func TestActionValid(t *testing.T) {
 	check := func(destType string, dest, dyn params) harness.QLResponse {
 		t.Helper()
 		var vars struct {
-			Input graphql2.ActionInput `json:"input"`
+			Input struct {
```

Comment:
should we be using `DestV1` here?

Reply:
No, since this is a test, it should be self-contained so that it fails if something changes in an incompatible way. Before, if ActionInput changed in a breaking way, this test wouldn't detect the problem since it re-used that type.